data/disconnect.json

```diff
@@ -10747,35 +10755,7 @@
             "ads-twitter.com"
           ]
         }
-      },
-      {
-        "Cnet": {
-          "https://www.cnet.com/": [
-            "2mdn.net"
-          ]
-        }
-      },
-      {
-        "Alexa": {
-          "http://www.alexa.com/": [
-            "pcache.alexa.com"
-          ]
-        }
-      },
-      {
-        "Saturn": {
-          "https://www.saturn.at/": [
-            "google-analytics.com"
-          ]
-        }
-      },
-      {
-        "Mediamarkt": {
-          "https://www.mediamarkt.at/": [
-            "google-analytics.com"
-          ]
-        }
```

Comment:
the two above failed on Android, I'm not aware did we have problems on desktop or not, we didn't on desktop iirc. The tests should be done on Android to verify that correctly. I don't know any ways how that can be done except put it to production and try or use a rooted phone and replace the file.

Reply:
I tested the mobile versions of the site (using the devtools), which is probably as much as we can do before going to production as you say.

I don't have a rooted phone to test this with. Maybe we should push to prod and then quickly test using a real Android phone?

Reply:
Sounds good and revert quickly back if we see problems on them.